<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExampleQueryDialog</class>
 <widget class="QDialog" name="AddExampleQueryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Example Query</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Query Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="queryNameEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
